### PR TITLE
Fix unexecuted test: "test_multiple_calls_present"

### PR DESCRIPTION
**Pull request by aranega**
*Issue*

Test `test_multiple_calls_present` is never executed because of a bad indent: the test is considered as an inner function of `test_show_one_call_present`.

*Description of changes:*

The `test_multiple_calls_present` indent is fixed to bring it back as a method instead of an inner class.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kdaily** [comment]
Thanks @aranega, checking on this!

File: tests/functional/history/test_list.py [hunk 1]
```diff
@@ -61,28 +61,28 @@ def test_show_one_call_present(self):
...
+                "Account": "bar",
+                "Arn": "arn:aws:iam::1234567:user/baz"
+            }
+        ]
+        _, _, rc = self.run_cmd('ec2 describe-instances', expected_rc=0)
+        self.history_recorder.record('CLI_RC', rc, 'CLI')
+        _, _, rc = self.run_cmd('sts get-caller-identity', expected_rc=0)
+        self.history_recorder.record('CLI_RC', rc, 'CLI')
+        self.run_cmd('history list', expected_rc=0)
+        self.assertIn(b'ec2 describe-regions',
```
Comment (kdaily) [inline tests/functional/history/test_list.py:85]: Looks like this should assert that `ec2 describe-instances` was called. Can you fix this (squash your commits, please)? I can assist as well!

**Review by kdaily — CHANGES_REQUESTED** [review]

File: tests/functional/history/test_list.py (hunk 1 above)
Comment (aranega) [reply tests/functional/history/test_list.py]: Hi @kdaily , sorry for the late answer, I didn't see the notification of your review. I made the modification and squashed the commits. I hope I didn't miss nothing.

**Review by nateprewitt — APPROVED** [review]
I made a couple minor tweaks to fix consistency in the tests and a layout issue. Otherwise, this looks good to go. Thanks @aranega, great find!

**Pull request merged**